### ISSUE TITLE
Latest episodes not saving after downlond

### ISSUE THREAD

**Issue by DrWho345** [gdavis/Mimir-Feedback#255]
### Are you sure this is a new issue?

- [X] I have confirmed this is a new issue

### macOS Version

macOS Monterey 12.7.3

### Your Mac

iMac (Retina 5K, 27 inch Late 2015)

### App Version

1.8.3(162)

### What happened?

Any attempt made to download new episodes of various podcasts is failing. I click download, and then have to click again, and the episodes are refusing to download/save

### Bug prevalence

just recently

### How does it happen?

1. Refresh podcast shows
2. Click download on new episodes
3. Progress circle appears
4. Upon finish, I go to my download folder and nothing has been updated, and try to download again with no luck

**Comment by gdavis** [comment]
Hello @DrWho345 sorry for the delayed response. Does this happen on all your podcast subscriptions or just one? If its just one, please let me know what podcast this is and I'll see if its an issue with the feed or content. 

Another thing to check: make sure Mimir has been given access to your downloads folder. You will initially get a popup asking you when you choose this as a save destination, but I wonder if that somehow got reset. You can check in the Settings app by searching for "Downloads" and see if Mimir has been allowed, as shown here:

<img width="718" alt="image" src="https://github.com/gdavis/Mimir-Feedback/assets/110691/913f799f-8ecb-46b7-9232-47d5570e530e">

**Comment by DrWho345** [comment]
Thank you for the reply, I coincidently was going to just write to you today, to finally put this hopefully to an end. It probably won't surprise you that I use other podcast downloader programs, Mac only, there are very few and all do something different, I was a devoted user of Downcast for years, until I upgraded to Monterey and it stopped being updated and working correctly, and had no choice but to find an alternative. I then discovered Podcast Archiver, which has been my life saver, and the only reason I came across Mimir, was the very, very few times Podcast Archiver let me down.

When Mimir started acting up, when I wrote to you originally I painstakingly moved, all the podcasts I had with Mimir to Podcast Archiver, and now use that exclusively, all except for one Podcast still on Mimir, which is Fangasm, Podcast Archiver shows it as blank, while Mimir works.

Today I loaded up Mimir, click download on the new episodes, it didn't work, and I thought I would just try to create a new download location, within my existing folder, that never had problems before... did that, clicked download and it worked, but now I have two folders, the existing one with 342 episodes, and another with 347 episodes.

Don't know if there is another solution for just making it one folder, but if this is the only way to keep it working then I guess we are good to go.

Thank you for your help.


________________________________
